encodings/BinaryEncoding.md

```diff
@@ -69,24 +69,29 @@ and use the default values.
 
 ### Trace Context
 
-#### Trace-id
+#### Fields added in Trace Context version 0
```

Comment:
What about removing "Fields added in" and just calling it "Trace Context version 0"?

Reply:
I wanted to show the full history of fields in this file, because it affects the serialization order.  Fields can be serialized in any order if they were added in the same version, but all fields added in an earlier version must precede all fields added in a later version.